query.go

```diff
@@ -24,11 +26,41 @@ func skippedQueryContext(_ context.Context, _ string, _ []driver.NamedValue) (dr
 	return nil, driver.ErrSkip
 }
 
+// add Custom Lables to metrics and traces
+type ctxKey string
+
+const (
+	meterCtxKey  ctxKey = "meterCtxKey"
+	tracerCtxKey ctxKey = "traceCtxKey"
```

Comment:
I dont think we need to separate tracer and meter attribute here. Why it is different between tracer and meter?

Reply:
Do you mean we should have 1 function to add attributes to both metrics and traces? We will loose the ability to set different custom attributes for meter and tracer, but that sounds okay for most cases.

Or should I remove it completely as it's not being used by us?

I added `AddTracerLabels` function just for the sake of completeness mostly.

Reply:
Lets not distinguish them. The attributes should be reusable for both.